spec/javascripts/govuk_publishing_components/rum/http-protocol-measurement-spec.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-env jasmine */
+/* global LUX */
+
+// This test will only work if the browser running it supports the HTML5
+// Performance API.
+if (typeof window.performance !== 'undefined') {
+  describe('The LUX RUM function should report the the HTTP protocol', function () {
+    // Because the script being tests is loaded and run as soon as possible, a
```

Comment:
Typo `being tests` should be `being tested`?

Reply:
Fixed - thank you!